lib/bas/serialize/github/issues.rb

```diff
@@ -3,10 +3,10 @@
 require_relative "../../domain/issue"
 require_relative "../base"
 
-module Mapper
+module Serialize
   module Github
     ##
-    # This class implementats the methods of the Mapper::Base module, specifically designed for
+    # This class implementats the methods of the Serialize::Base module, specifically designed for
```

Comment:
What do you think?, probably it is:
```suggestion
    # This class implements the methods of the Serialize::Base module, specifically designed for
```